src/components/Tokens/TokenTable/FavoriteButton.tsx

```diff
@@ -37,7 +37,7 @@ export default function FavoriteButton() {
   return (
     <StyledFavoriteButton onClick={() => setShowFavorites(!showFavorites)} active={showFavorites}>
       <FavoriteButtonContent>
-        <Heart size={17} color={theme.textPrimary} fill="transparent" />
+        <Heart size={17} color={showFavorites ? theme.white : theme.textPrimary} fill="transparent" />
```

Comment:
shouldn't there be a diff name for white?

Reply:
thought we tried to avoid using the actual colors instead of using the theme usually

Reply:
design wanted the text color to be 'white' for both light and dark mode (primary is black for light mode)